tests/test_prediction.py

```diff
@@ -93,6 +107,15 @@ def model_run(CLIENT):
         [f"[Pytest] Category Label ${i}" for i in range((len(TEST_IMG_URLS)))],
     )
 
+    response = ds.add_taxonomy(
+        "[Pytest] Scene Category Taxonomy 1",
+        "category",
+        [
+            f"[Pytest] Scene Category Label ${i}"
+            for i in range((len(TEST_IMG_URLS)))
+        ],
+    )
```

Comment:
Do we maybe want to extract this to a function? This seems to be copy pasted in a lot of places.